src/kv/RocksDBStore.cc

```diff
@@ -1172,7 +1172,13 @@ void RocksDBStore::compact_thread_entry()
       logger->set(l_rocksdb_compact_queue_len, compact_queue.size());
       compact_queue_lock.Unlock();
       logger->inc(l_rocksdb_compact_range);
-      compact_range(range.first, range.second);
+      if (range.first.empty() && range.second.empty()) {
+        derr << "Begin to compact leveldb store all..." << dendl;
```

Comment:
please drop this `derr` log or at least print it in `dout(20)`, as it's not an error.

Reply:
I just want to make the compact full event highlighted in the monitor log, because  sometimes we want  to know the compacting time. After all  it's triggered by ourselves.

Reply:
Yes, it's not an error, I think dout(0) is better.